### PR TITLE
Validation: May 6th. Fixed Alt-text and Invalid-author

### PR DESCRIPTION
**Pull request by Angelines-YG**
Fixed several alt-text missing issues and invalid-author

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=638) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=638) before we can accept your contribution.<br/><br/>:x: Angelines-YG [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=638)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetDocs?pullRequest=638) it.</sub>

File: aspnet/aspnet/overview/owin-and-katana/getting-started-with-owin-and-katana.md [hunk 1]
```diff
@@ -23,25 +23,25 @@ In this section, we'll host OWIN in IIS. This option gives you the flexibility a
...
+![Example illustration](getting-started-with-owin-and-katana/_static/image2.png)
 
 ### Add NuGet Packages
 
 Next, add the required NuGet packages. From the **Tools** menu, select **NuGet Package Manager**, then select **Package Manager Console**. In the Package Manager Console window, type the following command:
 
 `install-package Microsoft.Owin.Host.SystemWeb –Pre`
 
-![](getting-started-with-owin-and-katana/_static/image3.png)
+![Installing Package Manger, example illustration](getting-started-with-owin-and-katana/_static/image3.png)
```
Comment (kturetz) [inline aspnet/aspnet/overview/owin-and-katana/getting-started-with-owin-and-katana.md:38]: Should that be "Package Manager"?

File: aspnet/aspnet/overview/owin-and-katana/getting-started-with-owin-and-katana.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ Change the code in your `Startup1.Configuration` method as follows:
 
 Now use CTRL+F5 to run the application without debugging, so that Visual Studio will not break on the exception. The application behaves the same as before, until you navigate to `http://localhost/fail`, at which point the application throws the exception. The error page middleware will catch the exception and display an HTML page with information about the error. You can click the tabs to see the stack, query string, cookies, request header, and OWIN environment variables.
 
-![](getting-started-with-owin-and-katana/_static/image7.png)
+![Randome exeption image example](getting-started-with-owin-and-katana/_static/image7.png)
```
Comment (kturetz) [inline aspnet/aspnet/overview/owin-and-katana/getting-started-with-owin-and-katana.md:95]: Should that be "Random"?

File: aspnet/aspnet/overview/owin-and-katana/host-owin-in-an-azure-worker-role.md [hunk 3]
```diff
@@ -105,35 +105,35 @@ Build the solution, and press F5 to run the application locally in the Azure Com
...
 
 In Solution Explorer, right-click the AzureApp project. Select **Publish**.
 
-![](host-owin-in-an-azure-worker-role/_static/image12.png)
+![Publish AzureApp Illustration](host-owin-in-an-azure-worker-role/_static/image12.png)
 
 If you are not signed in to your Azure account, click **Sign In**.
 
-[![](host-owin-in-an-azure-worker-role/_static/image14.png)](host-owin-in-an-azure-worker-role/_static/image13.png)
+![Sign in example image](host-owin-in-an-azure-worker-role/_static/image13.png)
```
Comment (kturetz) [inline aspnet/aspnet/overview/owin-and-katana/host-owin-in-an-azure-worker-role.md:125]: "Sign-in"?

**Comment by Angelines-YG** [comment]
@Rick-Anderson 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge

File: aspnet/aspnet/overview/owin-and-katana/enabling-windows-authentication-in-katana.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 uid: aspnet/overview/owin-and-katana/enabling-windows-authentication-in-katana
 title: "Enabling Windows Authentication in Katana | Microsoft Docs"
-author: MikeWasson
+author: StephenBrentPeters
```
Comment (Rick-Anderson) [inline aspnet/aspnet/overview/owin-and-katana/enabling-windows-authentication-in-katana.md:4]:
```suggestion
author: rick-anderson
```

File: aspnet/aspnet/overview/owin-and-katana/getting-started-with-owin-and-katana.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
 uid: aspnet/overview/owin-and-katana/getting-started-with-owin-and-katana
 title: "Getting Started with OWIN and Katana | Microsoft Docs"
-author: MikeWasson
-description: ""
+author: StephenBrentPeters
```
Comment (Rick-Anderson) [inline aspnet/aspnet/overview/owin-and-katana/getting-started-with-owin-and-katana.md:4]: fix as previous

File: aspnet/aspnet/overview/owin-and-katana/host-owin-in-an-azure-worker-role.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 uid: aspnet/overview/owin-and-katana/host-owin-in-an-azure-worker-role
 title: "Host OWIN in an Azure Worker Role | Microsoft Docs"
-author: MikeWasson
+author: StephenBrentPeters
```
Comment (Rick-Anderson) [inline aspnet/aspnet/overview/owin-and-katana/host-owin-in-an-azure-worker-role.md:4]: fix

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Comment by Angelines-YG** [comment]
@Rick-Anderson 

Requested changes have been made. 
Please review within five business days and merge or comment in the PR with any additional changes you’d like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**